linux / python 2.7.3 smartcard/scard/__init__.py :  from smartcard.scard.scard import * ImportError: No module named scard

Hello,

I'm having some trouble with pyscard 1.9.1 installed via pip on an Intel Edison

running any example results in:

```
python pcsc_omnikey.py
Traceback (most recent call last):
  File "pcsc_omnikey.py", line 2, in <module>
    from smartcard.System import readers
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/__init__.py", line 32, in <module>
    from smartcard.System import listReaders
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/System.py", line 28, in <module>
    import smartcard.reader.ReaderFactory
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/reader/ReaderFactory.py", line 34, in <module>
    from smartcard.pcsc.PCSCReader import PCSCReader
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/pcsc/PCSCReader.py", line 28, in <module>
    from smartcard.pcsc.PCSCContext import PCSCContext
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/pcsc/PCSCContext.py", line 27, in <module>
    from smartcard.scard import *
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/scard/__init__.py", line 1, in <module>
    from smartcard.scard.scard import *
ImportError: No module named scard
```

If I modify smartcard/scard/__init__.py, changing it to

```
# from smartcard.sdcard.sdcard import * # original  
from smartcard.sdcard import * # modified
```

I get the following:

```
python pcsc_omnikey.py                            
Traceback (most recent call last):
  File "pcsc_omnikey.py", line 2, in <module>
    from smartcard.System import readers
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/__init__.py", line 32, in <module>
    from smartcard.System import listReaders
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/System.py", line 28, in <module>
    import smartcard.reader.ReaderFactory
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/reader/ReaderFactory.py", line 34, in <module>
    from smartcard.pcsc.PCSCReader import PCSCReader
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/pcsc/PCSCReader.py", line 29, in <module>
    from smartcard.pcsc.PCSCCardConnection import PCSCCardConnection
  File "/home/root/pyenv/lib/python2.7/site-packages/smartcard/pcsc/PCSCCardConnection.py", line 60, in <module>
    dictProtocolHeader = {SCARD_PCI_T0: 'T0', SCARD_PCI_T1: 'T1',
NameError: name 'SCARD_PCI_T0' is not defined
```

Unfortunately yes

```
python setup.py install
running install
running bdist_egg
running egg_info
writing requirements to pyscard.egg-info/requires.txt
writing pyscard.egg-info/PKG-INFO
writing top-level names to pyscard.egg-info/top_level.txt
writing dependency_links to pyscard.egg-info/dependency_links.txt
reading manifest file 'pyscard.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*.txt' under directory 'smartcard/Examples'
writing manifest file 'pyscard.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-i686/egg
running install_lib
running build_py
creating build
creating build/lib.linux-i686-2.7
creating build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardNames.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ExclusiveTransmitCardConnection.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/Observer.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/Synchronization.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardConnection.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ATR.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/Exceptions.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/AbstractCardRequest.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/__init__.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardRequest.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ulist.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/Session.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ReaderMonitoring.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardConnectionDecorator.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/Card.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardConnectionEvent.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardType.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ClassLoader.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/System.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardConnectionObserver.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardMonitoring.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/CardService.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/ExclusiveConnectCardConnection.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/PassThruCardService.py -> build/lib.linux-i686-2.7/smartcard
copying ./smartcard/guid.py -> build/lib.linux-i686-2.7/smartcard
creating build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCCardConnection.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/__init__.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCCardRequest.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCPart10.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCContext.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCReaderGroups.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCExceptions.py -> build/lib.linux-i686-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCReader.py -> build/lib.linux-i686-2.7/smartcard/pcsc
creating build/lib.linux-i686-2.7/smartcard/pyro
copying ./smartcard/pyro/__init__.py -> build/lib.linux-i686-2.7/smartcard/pyro
copying ./smartcard/pyro/PyroReader.py -> build/lib.linux-i686-2.7/smartcard/pyro
creating build/lib.linux-i686-2.7/smartcard/reader
copying ./smartcard/reader/ReaderFactory.py -> build/lib.linux-i686-2.7/smartcard/reader
copying ./smartcard/reader/__init__.py -> build/lib.linux-i686-2.7/smartcard/reader
copying ./smartcard/reader/ReaderGroups.py -> build/lib.linux-i686-2.7/smartcard/reader
copying ./smartcard/reader/Reader.py -> build/lib.linux-i686-2.7/smartcard/reader
creating build/lib.linux-i686-2.7/smartcard/scard
copying ./smartcard/scard/__init__.py -> build/lib.linux-i686-2.7/smartcard/scard
creating build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/SWExceptions.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_4ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_9ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/op21_ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/__init__.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_4_SW1ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_8ErrorChecker.py -> build/lib.linux-i686-2.7/smartcard/sw
copying ./smartcard/sw/ErrorCheckingChain.py -> build/lib.linux-i686-2.7/smartcard/sw
creating build/lib.linux-i686-2.7/smartcard/util
copying ./smartcard/util/__init__.py -> build/lib.linux-i686-2.7/smartcard/util
creating build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/APDUHexValidator.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/CardAndReaderTreePanel.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/APDUTracerPanel.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/__init__.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/ReaderToolbar.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardAppEventObserver.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardAppFrame.py -> build/lib.linux-i686-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardApp.py -> build/lib.linux-i686-2.7/smartcard/wx
running build_ext
building 'smartcard.scard._scard' extension
swigging smartcard/scard/scard.i to smartcard/scard/scard_wrap.c
swig -python -outdir smartcard/scard -DPCSCLITE -o smartcard/scard/scard_wrap.c smartcard/scard/scard.i
creating build/temp.linux-i686-2.7
creating build/temp.linux-i686-2.7/smartcard
creating build/temp.linux-i686-2.7/smartcard/scard
i586-poky-linux-gcc -m32 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mstackrealign -fno-omit-frame-pointer -fno-strict-aliasing -O2 -pipe -g -feliminate-unused-debug-types -DNDEBUG -g -O3 -Wall -Wstrict-prototypes -fPIC -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -Ismartcard/scard/ -I/usr/include/PCSC -I/usr/include/python2.7 -c smartcard/scard/helpers.c -o build/temp.linux-i686-2.7/smartcard/scard/helpers.o
i586-poky-linux-gcc -m32 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mstackrealign -fno-omit-frame-pointer -fno-strict-aliasing -O2 -pipe -g -feliminate-unused-debug-types -DNDEBUG -g -O3 -Wall -Wstrict-prototypes -fPIC -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -Ismartcard/scard/ -I/usr/include/PCSC -I/usr/include/python2.7 -c smartcard/scard/winscarddll.c -o build/temp.linux-i686-2.7/smartcard/scard/winscarddll.o
i586-poky-linux-gcc -m32 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mstackrealign -fno-omit-frame-pointer -fno-strict-aliasing -O2 -pipe -g -feliminate-unused-debug-types -DNDEBUG -g -O3 -Wall -Wstrict-prototypes -fPIC -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -Ismartcard/scard/ -I/usr/include/PCSC -I/usr/include/python2.7 -c smartcard/scard/scard_wrap.c -o build/temp.linux-i686-2.7/smartcard/scard/scard_wrap.o
In file included from smartcard/scard/scard_wrap.c:2991:0:
smartcard/scard/scard_wrap.c: In function '_wrap_SCardGetAttrib':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp3' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:3983:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg3 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp3' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:3998:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg3 );
         ^
smartcard/scard/scard_wrap.c: In function '_wrap_SCardStatus':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp5' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4888:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg5 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp5' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4924:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg5 );
         ^
smartcard/scard/scard_wrap.c: In function '_wrap_SCardListReaderGroups':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp2' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4685:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg2 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp2' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4714:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg2 );
         ^
smartcard/scard/scard_wrap.c: In function '_wrap_SCardListReaders':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp3' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4569:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg3 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp3' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4625:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg3 );
         ^
smartcard/scard/scard_wrap.c: In function '_wrap_SCardControl':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp4' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4138:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg4 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp4' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:4166:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg4 );
         ^
smartcard/scard/scard_wrap.c: In function '_wrap_SCardTransmit':
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp4' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:5000:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg4 );
         ^
smartcard/scard/memlog.h:50:26: warning: attempt to free a non-heap object 'temp4' [-Wfree-nonheap-object]
         #define mem_Free free
                          ^
smartcard/scard/scard_wrap.c:5028:9: note: in expansion of macro 'mem_Free'
         mem_Free( arg4 );
         ^
i586-poky-linux-gcc -m32 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mstackrealign -fno-omit-frame-pointer -shared -Wl,-O1 -Wl,--hash-style=gnu -Wl,--as-needed build/temp.linux-i686-2.7/smartcard/scard/helpers.o build/temp.linux-i686-2.7/smartcard/scard/winscarddll.o build/temp.linux-i686-2.7/smartcard/scard/scard_wrap.o -o build/lib.linux-i686-2.7/smartcard/scard/_scard.so
creating build/bdist.linux-i686
creating build/bdist.linux-i686/egg
creating build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardNames.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ExclusiveTransmitCardConnection.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/pyro
copying build/lib.linux-i686-2.7/smartcard/pyro/__init__.py -> build/bdist.linux-i686/egg/smartcard/pyro
copying build/lib.linux-i686-2.7/smartcard/pyro/PyroReader.py -> build/bdist.linux-i686/egg/smartcard/pyro
creating build/bdist.linux-i686/egg/smartcard/scard
copying build/lib.linux-i686-2.7/smartcard/scard/__init__.py -> build/bdist.linux-i686/egg/smartcard/scard
copying build/lib.linux-i686-2.7/smartcard/scard/_scard.so -> build/bdist.linux-i686/egg/smartcard/scard
copying build/lib.linux-i686-2.7/smartcard/Observer.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCCardConnection.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/__init__.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCCardRequest.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCPart10.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCContext.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCReaderGroups.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCExceptions.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/pcsc/PCSCReader.py -> build/bdist.linux-i686/egg/smartcard/pcsc
copying build/lib.linux-i686-2.7/smartcard/Synchronization.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardConnection.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ATR.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/Exceptions.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/AbstractCardRequest.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/__init__.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardRequest.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ulist.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/Session.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ReaderMonitoring.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/reader
copying build/lib.linux-i686-2.7/smartcard/reader/ReaderFactory.py -> build/bdist.linux-i686/egg/smartcard/reader
copying build/lib.linux-i686-2.7/smartcard/reader/__init__.py -> build/bdist.linux-i686/egg/smartcard/reader
copying build/lib.linux-i686-2.7/smartcard/reader/ReaderGroups.py -> build/bdist.linux-i686/egg/smartcard/reader
copying build/lib.linux-i686-2.7/smartcard/reader/Reader.py -> build/bdist.linux-i686/egg/smartcard/reader
copying build/lib.linux-i686-2.7/smartcard/CardConnectionDecorator.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/APDUHexValidator.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/CardAndReaderTreePanel.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/APDUTracerPanel.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/__init__.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/ReaderToolbar.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/SimpleSCardAppEventObserver.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/SimpleSCardAppFrame.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/wx/SimpleSCardApp.py -> build/bdist.linux-i686/egg/smartcard/wx
copying build/lib.linux-i686-2.7/smartcard/Card.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardConnectionEvent.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardType.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ClassLoader.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/System.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardConnectionObserver.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/CardMonitoring.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/SWExceptions.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ISO7816_4ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ISO7816_9ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/op21_ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/__init__.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ISO7816_4_SW1ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ISO7816_8ErrorChecker.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/sw/ErrorCheckingChain.py -> build/bdist.linux-i686/egg/smartcard/sw
copying build/lib.linux-i686-2.7/smartcard/CardService.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/ExclusiveConnectCardConnection.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/PassThruCardService.py -> build/bdist.linux-i686/egg/smartcard
copying build/lib.linux-i686-2.7/smartcard/guid.py -> build/bdist.linux-i686/egg/smartcard
creating build/bdist.linux-i686/egg/smartcard/util
copying build/lib.linux-i686-2.7/smartcard/util/__init__.py -> build/bdist.linux-i686/egg/smartcard/util
byte-compiling build/bdist.linux-i686/egg/smartcard/CardNames.py to CardNames.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ExclusiveTransmitCardConnection.py to ExclusiveTransmitCardConnection.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pyro/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pyro/PyroReader.py to PyroReader.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/scard/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/Observer.py to Observer.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCCardConnection.py to PCSCCardConnection.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCCardRequest.py to PCSCCardRequest.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCPart10.py to PCSCPart10.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCContext.py to PCSCContext.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCReaderGroups.py to PCSCReaderGroups.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCExceptions.py to PCSCExceptions.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/pcsc/PCSCReader.py to PCSCReader.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/Synchronization.py to Synchronization.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardConnection.py to CardConnection.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ATR.py to ATR.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/Exceptions.py to Exceptions.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/AbstractCardRequest.py to AbstractCardRequest.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardRequest.py to CardRequest.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ulist.py to ulist.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/Session.py to Session.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ReaderMonitoring.py to ReaderMonitoring.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/reader/ReaderFactory.py to ReaderFactory.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/reader/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/reader/ReaderGroups.py to ReaderGroups.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/reader/Reader.py to Reader.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardConnectionDecorator.py to CardConnectionDecorator.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/APDUHexValidator.py to APDUHexValidator.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/CardAndReaderTreePanel.py to CardAndReaderTreePanel.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/APDUTracerPanel.py to APDUTracerPanel.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/ReaderToolbar.py to ReaderToolbar.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/SimpleSCardAppEventObserver.py to SimpleSCardAppEventObserver.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/SimpleSCardAppFrame.py to SimpleSCardAppFrame.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/wx/SimpleSCardApp.py to SimpleSCardApp.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/Card.py to Card.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardConnectionEvent.py to CardConnectionEvent.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardType.py to CardType.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ClassLoader.py to ClassLoader.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/System.py to System.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardConnectionObserver.py to CardConnectionObserver.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardMonitoring.py to CardMonitoring.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/SWExceptions.py to SWExceptions.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ISO7816_4ErrorChecker.py to ISO7816_4ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ISO7816_9ErrorChecker.py to ISO7816_9ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/op21_ErrorChecker.py to op21_ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/__init__.py to __init__.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ErrorChecker.py to ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ISO7816_4_SW1ErrorChecker.py to ISO7816_4_SW1ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ISO7816_8ErrorChecker.py to ISO7816_8ErrorChecker.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/sw/ErrorCheckingChain.py to ErrorCheckingChain.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/CardService.py to CardService.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/ExclusiveConnectCardConnection.py to ExclusiveConnectCardConnection.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/PassThruCardService.py to PassThruCardService.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/guid.py to guid.pyc
byte-compiling build/bdist.linux-i686/egg/smartcard/util/__init__.py to __init__.pyc
creating stub loader for smartcard/scard/_scard.so
byte-compiling build/bdist.linux-i686/egg/smartcard/scard/_scard.py to _scard.pyc
creating build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/PKG-INFO -> build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/SOURCES.txt -> build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/dependency_links.txt -> build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/pbr.json -> build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/requires.txt -> build/bdist.linux-i686/egg/EGG-INFO
copying pyscard.egg-info/top_level.txt -> build/bdist.linux-i686/egg/EGG-INFO
writing build/bdist.linux-i686/egg/EGG-INFO/native_libs.txt
zip_safe flag not set; analyzing archive contents...
smartcard.wx.__init__: module references __file__
creating dist
creating 'dist/pyscard-1.9.1-py2.7-linux-i686.egg' and adding 'build/bdist.linux-i686/egg' to it
removing 'build/bdist.linux-i686/egg' (and everything under it)
Processing pyscard-1.9.1-py2.7-linux-i686.egg
creating /home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg
Extracting pyscard-1.9.1-py2.7-linux-i686.egg to /home/root/pyenv/lib/python2.7/site-packages
Adding pyscard 1.9.1 to easy-install.pth file

Installed /home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg
Processing dependencies for pyscard==1.9.1
Finished processing dependencies for pyscard==1.9.1
```

```
 python
Python 2.7.3 (default, Jun 19 2015, 07:08:38)
[GCC 4.9.1] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import smartcard
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/__init__.py", line 32, in <module>
    from smartcard.System import listReaders
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/System.py", line 28, in <module>
    import smartcard.reader.ReaderFactory
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/reader/ReaderFactory.py", line 34, in <module>
    from smartcard.pcsc.PCSCReader import PCSCReader
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/pcsc/PCSCReader.py", line 28, in <module>
    from smartcard.pcsc.PCSCContext import PCSCContext
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/pcsc/PCSCContext.py", line 27, in <module>
    from smartcard.scard import *
  File "/home/root/pyenv/lib/python2.7/site-packages/pyscard-1.9.1-py2.7-linux-i686.egg/smartcard/scard/__init__.py", line 1, in <module>
    from smartcard.scard.scard import *
ImportError: No module named scard
>>>
```

I can also note that trying earlier versions e.g. 1.6.* results in the second error above (name 'SCARD_PCI_T0' is not defined)

(I  suspect this is somehow a swig issue)

@LudovicRousseau, apologies, all commands execute fine and error is exactly the same as the others hence I did not copy them but here they are.

```
sudo pip uninstall pyscard
Uninstalling pyscard-1.9.1:
  /Library/Python/2.7/site-packages/pyscard-1.9.1-py2.7-macosx-10.11-intel.egg
Proceed (y/n)? y
Successfully uninstalled pyscard-1.9.1
```

second time:
```
sudo pip uninstall pyscard
Cannot uninstall requirement pyscard, not installed
```

cleanup
```
sudo rm -rf pyscard/
```
download
```
git clone https://github.com/LudovicRousseau/pyscard.git
Cloning into 'pyscard'...
remote: Counting objects: 5677, done.
remote: Total 5677 (delta 0), reused 0 (delta 0), pack-reused 5677
Receiving objects: 100% (5677/5677), 1000.95 KiB | 709.00 KiB/s, done.
Resolving deltas: 100% (4294/4294), done.
Checking connectivity... done.

```
install
```
cd pyscard/
sudo python setup.py install
running install
Checking .pth file support in /Library/Python/2.7/site-packages/
/usr/bin/python -E -c pass
TEST PASSED: /Library/Python/2.7/site-packages/ appears to support .pth files
running bdist_egg
running egg_info
creating pyscard.egg-info
writing requirements to pyscard.egg-info/requires.txt
writing pyscard.egg-info/PKG-INFO
writing top-level names to pyscard.egg-info/top_level.txt
writing dependency_links to pyscard.egg-info/dependency_links.txt
writing manifest file 'pyscard.egg-info/SOURCES.txt'
reading manifest file 'pyscard.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching '*.txt' under directory 'smartcard/Examples'
warning: no files found matching '*.html' under directory 'smartcard/doc'
warning: no files found matching '*.css' under directory 'smartcard/doc'
warning: no files found matching '*.js' under directory 'smartcard/doc'
warning: no files found matching '*.txt' under directory 'smartcard/doc'
writing manifest file 'pyscard.egg-info/SOURCES.txt'
installing library code to build/bdist.macosx-10.11-intel/egg
running install_lib
running build_py
creating build
creating build/lib.macosx-10.11-intel-2.7
creating build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/AbstractCardRequest.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ATR.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/Card.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardConnection.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardConnectionDecorator.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardConnectionEvent.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardConnectionObserver.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardMonitoring.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardNames.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardRequest.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardService.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/CardType.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ClassLoader.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/Exceptions.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ExclusiveConnectCardConnection.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ExclusiveTransmitCardConnection.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/guid.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/Observer.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/PassThruCardService.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ReaderMonitoring.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/Session.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/Synchronization.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/System.py -> build/lib.macosx-10.11-intel-2.7/smartcard
copying ./smartcard/ulist.py -> build/lib.macosx-10.11-intel-2.7/smartcard
creating build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCCardConnection.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCCardRequest.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCContext.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCExceptions.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCPart10.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCReader.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
copying ./smartcard/pcsc/PCSCReaderGroups.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pcsc
creating build/lib.macosx-10.11-intel-2.7/smartcard/pyro
copying ./smartcard/pyro/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pyro
copying ./smartcard/pyro/PyroReader.py -> build/lib.macosx-10.11-intel-2.7/smartcard/pyro
creating build/lib.macosx-10.11-intel-2.7/smartcard/reader
copying ./smartcard/reader/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/reader
copying ./smartcard/reader/Reader.py -> build/lib.macosx-10.11-intel-2.7/smartcard/reader
copying ./smartcard/reader/ReaderFactory.py -> build/lib.macosx-10.11-intel-2.7/smartcard/reader
copying ./smartcard/reader/ReaderGroups.py -> build/lib.macosx-10.11-intel-2.7/smartcard/reader
creating build/lib.macosx-10.11-intel-2.7/smartcard/scard
copying ./smartcard/scard/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/scard
creating build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ErrorCheckingChain.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_4_SW1ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_4ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_8ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/ISO7816_9ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/op21_ErrorChecker.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
copying ./smartcard/sw/SWExceptions.py -> build/lib.macosx-10.11-intel-2.7/smartcard/sw
creating build/lib.macosx-10.11-intel-2.7/smartcard/util
copying ./smartcard/util/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/util
creating build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/__init__.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/APDUHexValidator.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/APDUTracerPanel.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/CardAndReaderTreePanel.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/ReaderToolbar.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardApp.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardAppEventObserver.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
copying ./smartcard/wx/SimpleSCardAppFrame.py -> build/lib.macosx-10.11-intel-2.7/smartcard/wx
running build_ext
building 'smartcard.scard._scard' extension
swigging smartcard/scard/scard.i to smartcard/scard/scard_wrap.c
swig -python -outdir smartcard/scard -DPCSCLITE -D__APPLE__ -D__LION__ -o smartcard/scard/scard_wrap.c smartcard/scard/scard.i
creating build/temp.macosx-10.11-intel-2.7
creating build/temp.macosx-10.11-intel-2.7/smartcard
creating build/temp.macosx-10.11-intel-2.7/smartcard/scard
cc -fno-strict-aliasing -fno-common -dynamic -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -pipe -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -D__APPLE__=1 -D__LION__=1 -Ismartcard/scard/ -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c smartcard/scard/helpers.c -o build/temp.macosx-10.11-intel-2.7/smartcard/scard/helpers.o -v -arch i386 -arch x86_64 -ggdb
Apple LLVM version 7.0.2 (clang-700.1.81)
Target: x86_64-apple-darwin15.2.0
Thread model: posix
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple i386-apple-macosx10.11.0 -emit-obj -disable-free -disable-llvm-verifier -main-file-name helpers.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -target-cpu yonah -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/helpers.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-fragile-10.11.0 -fobjc-subscripting-legacy-runtime -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/helpers-922dc8.o -x c smartcard/scard/helpers.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx10.11.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -emit-obj -disable-free -disable-llvm-verifier -main-file-name helpers.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -munwind-tables -target-cpu core2 -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/helpers.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-10.11.0 -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/helpers-5ed952.o -x c smartcard/scard/helpers.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
smartcard/scard/helpers.c:808:17: warning: implicit conversion loses integer precision: 'long' to 'SCARDCONTEXT' (aka 'int') [-Wshorten-64-to-32]
        scRet = PyLong_AsLong( source );
              ~ ^~~~~~~~~~~~~~~~~~~~~~~
smartcard/scard/helpers.c:878:17: warning: implicit conversion loses integer precision: 'long' to 'SCARDHANDLE' (aka 'int') [-Wshorten-64-to-32]
        scRet = PyLong_AsLong( source );
              ~ ^~~~~~~~~~~~~~~~~~~~~~~
smartcard/scard/helpers.c:947:17: warning: implicit conversion loses integer precision: 'long' to 'SCARDDWORDARG' (aka 'unsigned int') [-Wshorten-64-to-32]
        scRet = PyLong_AsLong( source );
              ~ ^~~~~~~~~~~~~~~~~~~~~~~
3 warnings generated.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo" -create -output build/temp.macosx-10.11-intel-2.7/smartcard/scard/helpers.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/helpers-922dc8.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/helpers-5ed952.o
cc -fno-strict-aliasing -fno-common -dynamic -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -pipe -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -D__APPLE__=1 -D__LION__=1 -Ismartcard/scard/ -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c smartcard/scard/winscarddll.c -o build/temp.macosx-10.11-intel-2.7/smartcard/scard/winscarddll.o -v -arch i386 -arch x86_64 -ggdb
Apple LLVM version 7.0.2 (clang-700.1.81)
Target: x86_64-apple-darwin15.2.0
Thread model: posix
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple i386-apple-macosx10.11.0 -emit-obj -disable-free -disable-llvm-verifier -main-file-name winscarddll.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -target-cpu yonah -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/winscarddll.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-fragile-10.11.0 -fobjc-subscripting-legacy-runtime -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/winscarddll-a047dd.o -x c smartcard/scard/winscarddll.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx10.11.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -emit-obj -disable-free -disable-llvm-verifier -main-file-name winscarddll.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -munwind-tables -target-cpu core2 -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/winscarddll.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-10.11.0 -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/winscarddll-7862e7.o -x c smartcard/scard/winscarddll.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo" -create -output build/temp.macosx-10.11-intel-2.7/smartcard/scard/winscarddll.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/winscarddll-a047dd.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/winscarddll-7862e7.o
cc -fno-strict-aliasing -fno-common -dynamic -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -pipe -DVER_PRODUCTVERSION=1,9,1,0000 -DVER_PRODUCTVERSION_STR=1.9.1 -DPCSCLITE=1 -D__APPLE__=1 -D__LION__=1 -Ismartcard/scard/ -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c smartcard/scard/scard_wrap.c -o build/temp.macosx-10.11-intel-2.7/smartcard/scard/scard_wrap.o -v -arch i386 -arch x86_64 -ggdb
Apple LLVM version 7.0.2 (clang-700.1.81)
Target: x86_64-apple-darwin15.2.0
Thread model: posix
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple i386-apple-macosx10.11.0 -emit-obj -disable-free -disable-llvm-verifier -main-file-name scard_wrap.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -target-cpu yonah -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/scard_wrap.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-fragile-10.11.0 -fobjc-subscripting-legacy-runtime -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/scard_wrap-ba0cd4.o -x c smartcard/scard/scard_wrap.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" -cc1 -triple x86_64-apple-macosx10.11.0 -Wdeprecated-objc-isa-usage -Werror=deprecated-objc-isa-usage -emit-obj -disable-free -disable-llvm-verifier -main-file-name scard_wrap.c -mrelocation-model pic -pic-level 2 -mthread-model posix -mdisable-fp-elim -relaxed-aliasing -masm-verbose -munwind-tables -target-cpu core2 -target-linker-version 253.9 -v -gdwarf-2 -dwarf-column-info -coverage-file /Users/mahagen/Downloads/EMVCAP-1.4/pyscard/build/temp.macosx-10.11-intel-2.7/smartcard/scard/scard_wrap.o -resource-dir /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -D ENABLE_DTRACE -D MACOSX -D NDEBUG -D NDEBUG -D ENABLE_DTRACE -D VER_PRODUCTVERSION=1,9,1,0000 -D VER_PRODUCTVERSION_STR=1.9.1 -D PCSCLITE=1 -D __APPLE__=1 -D __LION__=1 -I smartcard/scard/ -I /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -Os -Wall -Wstrict-prototypes -Wshorten-64-to-32 -Wall -Wstrict-prototypes -fdebug-compilation-dir /Users/mahagen/Downloads/EMVCAP-1.4/pyscard -ferror-limit 19 -fmessage-length 195 -fwrapv -stack-protector 1 -mstackrealign -fblocks -fobjc-runtime=macosx-10.11.0 -fencode-extended-block-signature -fmax-type-align=16 -fno-common -fdiagnostics-show-option -fcolor-diagnostics -vectorize-loops -vectorize-slp -o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/scard_wrap-501733.o -x c smartcard/scard/scard_wrap.c
clang -cc1 version 7.0.2 based upon LLVM 3.7.0svn default target x86_64-apple-darwin15.2.0
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/local/include"
ignoring nonexistent directory "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/Library/Frameworks"
#include "..." search starts here:
#include <...> search starts here:
 smartcard/scard
 /System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.0.2/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
End of search list.
smartcard/scard/scard_wrap.c:3213:16: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
        return lRetCode;
        ~~~~~~ ^~~~~~~~
smartcard/scard/scard_wrap.c:3223:12: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
    return lRetCode;
    ~~~~~~ ^~~~~~~~
smartcard/scard/scard_wrap.c:3232:12: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
    return lRetCode;
    ~~~~~~ ^~~~~~~~
smartcard/scard/scard_wrap.c:3523:12: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
    return lRetCode;
    ~~~~~~ ^~~~~~~~
smartcard/scard/scard_wrap.c:3568:12: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
    return ret;
    ~~~~~~ ^~~
smartcard/scard/scard_wrap.c:3611:51: warning: implicit conversion loses integer precision: 'long' to 'SCARDRETCODE' (aka 'int') [-Wshorten-64-to-32]
        return (ERRORSTRING*)myPcscStringifyError(lErrCode);
                             ~~~~~~~~~~~~~~~~~~~~ ^~~~~~~~
6 warnings generated.
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lipo" -create -output build/temp.macosx-10.11-intel-2.7/smartcard/scard/scard_wrap.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/scard_wrap-ba0cd4.o /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/scard_wrap-501733.o
cc -bundle -undefined dynamic_lookup -Wl,-F. build/temp.macosx-10.11-intel-2.7/smartcard/scard/helpers.o build/temp.macosx-10.11-intel-2.7/smartcard/scard/winscarddll.o build/temp.macosx-10.11-intel-2.7/smartcard/scard/scard_wrap.o -o build/lib.macosx-10.11-intel-2.7/smartcard/scard/_scard.so -arch i386 -arch x86_64 -ggdb
creating build/bdist.macosx-10.11-intel
creating build/bdist.macosx-10.11-intel/egg
creating build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/AbstractCardRequest.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/ATR.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/Card.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardConnection.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardConnectionDecorator.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardConnectionEvent.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardConnectionObserver.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardMonitoring.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardNames.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardRequest.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardService.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/CardType.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/ClassLoader.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/Exceptions.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/ExclusiveConnectCardConnection.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/ExclusiveTransmitCardConnection.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/guid.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/Observer.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/PassThruCardService.py -> build/bdist.macosx-10.11-intel/egg/smartcard
creating build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCCardConnection.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCCardRequest.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCContext.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCExceptions.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCPart10.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCReader.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
copying build/lib.macosx-10.11-intel-2.7/smartcard/pcsc/PCSCReaderGroups.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pcsc
creating build/bdist.macosx-10.11-intel/egg/smartcard/pyro
copying build/lib.macosx-10.11-intel-2.7/smartcard/pyro/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pyro
copying build/lib.macosx-10.11-intel-2.7/smartcard/pyro/PyroReader.py -> build/bdist.macosx-10.11-intel/egg/smartcard/pyro
creating build/bdist.macosx-10.11-intel/egg/smartcard/reader
copying build/lib.macosx-10.11-intel-2.7/smartcard/reader/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/reader
copying build/lib.macosx-10.11-intel-2.7/smartcard/reader/Reader.py -> build/bdist.macosx-10.11-intel/egg/smartcard/reader
copying build/lib.macosx-10.11-intel-2.7/smartcard/reader/ReaderFactory.py -> build/bdist.macosx-10.11-intel/egg/smartcard/reader
copying build/lib.macosx-10.11-intel-2.7/smartcard/reader/ReaderGroups.py -> build/bdist.macosx-10.11-intel/egg/smartcard/reader
copying build/lib.macosx-10.11-intel-2.7/smartcard/ReaderMonitoring.py -> build/bdist.macosx-10.11-intel/egg/smartcard
creating build/bdist.macosx-10.11-intel/egg/smartcard/scard
copying build/lib.macosx-10.11-intel-2.7/smartcard/scard/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/scard
copying build/lib.macosx-10.11-intel-2.7/smartcard/scard/_scard.so -> build/bdist.macosx-10.11-intel/egg/smartcard/scard
copying build/lib.macosx-10.11-intel-2.7/smartcard/Session.py -> build/bdist.macosx-10.11-intel/egg/smartcard
creating build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ErrorCheckingChain.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ISO7816_4_SW1ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ISO7816_4ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ISO7816_8ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/ISO7816_9ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/op21_ErrorChecker.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/sw/SWExceptions.py -> build/bdist.macosx-10.11-intel/egg/smartcard/sw
copying build/lib.macosx-10.11-intel-2.7/smartcard/Synchronization.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/System.py -> build/bdist.macosx-10.11-intel/egg/smartcard
copying build/lib.macosx-10.11-intel-2.7/smartcard/ulist.py -> build/bdist.macosx-10.11-intel/egg/smartcard
creating build/bdist.macosx-10.11-intel/egg/smartcard/util
copying build/lib.macosx-10.11-intel-2.7/smartcard/util/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/util
creating build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/__init__.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/APDUHexValidator.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/APDUTracerPanel.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/CardAndReaderTreePanel.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/ReaderToolbar.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/SimpleSCardApp.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/SimpleSCardAppEventObserver.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
copying build/lib.macosx-10.11-intel-2.7/smartcard/wx/SimpleSCardAppFrame.py -> build/bdist.macosx-10.11-intel/egg/smartcard/wx
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/AbstractCardRequest.py to AbstractCardRequest.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ATR.py to ATR.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/Card.py to Card.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardConnection.py to CardConnection.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardConnectionDecorator.py to CardConnectionDecorator.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardConnectionEvent.py to CardConnectionEvent.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardConnectionObserver.py to CardConnectionObserver.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardMonitoring.py to CardMonitoring.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardNames.py to CardNames.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardRequest.py to CardRequest.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardService.py to CardService.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/CardType.py to CardType.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ClassLoader.py to ClassLoader.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/Exceptions.py to Exceptions.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ExclusiveConnectCardConnection.py to ExclusiveConnectCardConnection.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ExclusiveTransmitCardConnection.py to ExclusiveTransmitCardConnection.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/guid.py to guid.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/Observer.py to Observer.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/PassThruCardService.py to PassThruCardService.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCCardConnection.py to PCSCCardConnection.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCCardRequest.py to PCSCCardRequest.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCContext.py to PCSCContext.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCExceptions.py to PCSCExceptions.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCPart10.py to PCSCPart10.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCReader.py to PCSCReader.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pcsc/PCSCReaderGroups.py to PCSCReaderGroups.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pyro/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/pyro/PyroReader.py to PyroReader.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/reader/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/reader/Reader.py to Reader.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/reader/ReaderFactory.py to ReaderFactory.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/reader/ReaderGroups.py to ReaderGroups.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ReaderMonitoring.py to ReaderMonitoring.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/scard/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/Session.py to Session.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ErrorChecker.py to ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ErrorCheckingChain.py to ErrorCheckingChain.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ISO7816_4_SW1ErrorChecker.py to ISO7816_4_SW1ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ISO7816_4ErrorChecker.py to ISO7816_4ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ISO7816_8ErrorChecker.py to ISO7816_8ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/ISO7816_9ErrorChecker.py to ISO7816_9ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/op21_ErrorChecker.py to op21_ErrorChecker.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/sw/SWExceptions.py to SWExceptions.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/Synchronization.py to Synchronization.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/System.py to System.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/ulist.py to ulist.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/util/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/APDUHexValidator.py to APDUHexValidator.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/APDUTracerPanel.py to APDUTracerPanel.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/CardAndReaderTreePanel.py to CardAndReaderTreePanel.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/ReaderToolbar.py to ReaderToolbar.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/SimpleSCardApp.py to SimpleSCardApp.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/SimpleSCardAppEventObserver.py to SimpleSCardAppEventObserver.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/wx/SimpleSCardAppFrame.py to SimpleSCardAppFrame.pyc
creating stub loader for smartcard/scard/_scard.so
byte-compiling build/bdist.macosx-10.11-intel/egg/smartcard/scard/_scard.py to _scard.pyc
creating build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying pyscard.egg-info/PKG-INFO -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying pyscard.egg-info/SOURCES.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying pyscard.egg-info/dependency_links.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying pyscard.egg-info/requires.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying pyscard.egg-info/top_level.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
writing build/bdist.macosx-10.11-intel/egg/EGG-INFO/native_libs.txt
zip_safe flag not set; analyzing archive contents...
smartcard.wx.__init__: module references __file__
creating dist
creating 'dist/pyscard-1.9.1-py2.7-macosx-10.11-intel.egg' and adding 'build/bdist.macosx-10.11-intel/egg' to it
removing 'build/bdist.macosx-10.11-intel/egg' (and everything under it)
Processing pyscard-1.9.1-py2.7-macosx-10.11-intel.egg
creating /Library/Python/2.7/site-packages/pyscard-1.9.1-py2.7-macosx-10.11-intel.egg
Extracting pyscard-1.9.1-py2.7-macosx-10.11-intel.egg to /Library/Python/2.7/site-packages
Adding pyscard 1.9.1 to easy-install.pth file

Installed /Library/Python/2.7/site-packages/pyscard-1.9.1-py2.7-macosx-10.11-intel.egg
Processing dependencies for pyscard==1.9.1
Finished processing dependencies for pyscard==1.9.1
```

Error 
```
Python 2.7.10 (default, Oct 23 2015, 18:05:06) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import smartcard
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "smartcard/__init__.py", line 32, in <module>
    from smartcard.System import listReaders
  File "smartcard/System.py", line 28, in <module>
    import smartcard.reader.ReaderFactory
  File "smartcard/reader/ReaderFactory.py", line 34, in <module>
    from smartcard.pcsc.PCSCReader import PCSCReader
  File "smartcard/pcsc/PCSCReader.py", line 28, in <module>
    from smartcard.pcsc.PCSCContext import PCSCContext
  File "smartcard/pcsc/PCSCContext.py", line 27, in <module>
    from smartcard.scard import *
  File "smartcard/scard/__init__.py", line 1, in <module>
    from smartcard.scard.scard import *
  File "smartcard/scard/scard.py", line 129, in <module>
    _scard = swig_import_helper()
  File "smartcard/scard/scard.py", line 121, in swig_import_helper
    import _scard
ImportError: No module named _scard
```

lib
```
ls /Library/Python/2.7/site-packages/pyscard-1.9.1-py2.7-macosx-10.11-intel.egg/smartcard/scard/
__init__.py	
__init__.pyc	
_scard.py	
_scard.pyc	
_scard.so
```

Did I miss anything?

Thank you ....

--- a/setup.py
+++ b/setup.py
@@ -24,9 +24,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 """
 
 from distutils.util import get_platform
+import distutils.command.install as install_orig
+import inspect
 import sys
 
 from setuptools import setup, Extension
+from setuptools.command.bdist_egg import bdist_egg
+from setuptools.command.install import install
 
 
 if sys.version_info[0:2] < (2, 6):
@@ -83,6 +87,35 @@ VERSION_STR = '%i.%i.%i' % VERSION_INFO[:3]
 VERSION_ALT = '%i,%01i,%01i,%04i' % VERSION_INFO
 
 
+# Workaround for `pip install` and direct `setup.py install`
+class InstallBuildExtFirst(install):
+    """Workaround substitute `install` command"""
+    def run(self):
+        # Run built_ext first so that SWIG generated files are included
+        self.run_command("build_ext")
+        # Copy the rest of the logic from setuptools install so that the
+        # stack frame logic is preserved
+
+        # Explicit request for old-style install?  Just do it
+        if self.old_and_unmanageable or self.single_version_externally_managed:
+            return install_orig.install.run(self)
+
+        if not self._called_from_setup(inspect.currentframe()):
+            # Run in backward-compatibility mode to support bdist_* commands.
+            install_orig.install.run(self)
+        else:
+            self.do_egg_install()
+
+
+# Workaround for `easy_install`
+class BdistEggBuildExtFirst(bdist_egg):
+    """Workaround substitute `bdist_egg` command"""
+    def run(self):
+        # Run build_ext first so that SWIG generated files are included
+        self.run_command("build_ext")
+        return bdist_egg.run(self)
+
+
 kw = {'name': "pyscard",
       'version': VERSION_STR,
       'description': "Smartcard module for Python.",
@@ -106,6 +139,10 @@ kw = {'name': "pyscard",
                          "smartcard/wx": ["resources/*.ico"],
                        },
 
+      'cmdclass': {
+          'install': InstallBuildExtFirst,
+          'bdist_egg': BdistEggBuildExtFirst,
+      },
       # the _scard.pyd extension to build
       'ext_modules': [Extension("smartcard.scard._scard",
                              define_macros=[

